test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/Output/AttributeTargetingTagHelpers.DesignTime.cs

```diff
@@ -20,10 +20,10 @@ private void @__RazorDesignTimeHelpers__()
             #pragma warning restore 219
         }
         #line hidden
-        private PTagHelper __PTagHelper = null;
-        private CatchAllTagHelper __CatchAllTagHelper = null;
-        private InputTagHelper __InputTagHelper = null;
-        private InputTagHelper2 __InputTagHelper2 = null;
+        private global::PTagHelper __PTagHelper = null;
```

Comment:
Need to expand the `TypeName` properties of the test `TagHelperDescriptor` to make it clear these types are written correctly. Looks Just Wrong:tm: as-is.